src/fonts/load_all.ts

```diff
@@ -15,6 +16,7 @@ import { loadPetaluma } from './load_petaluma';
 export function loadAllMusicFonts(): void {
   loadBravura();
   loadGonville();
+  loadGootville();
```

Comment:
Are people OK with the main `vexflow.js` library growing bigger as we add more SMuFL fonts? We do have targets to generate builds with a single font, but I'm wondering if we want the main library to include all fonts.

@0xfe @mscuthbert @AaronDavidNewman

Reply:
This makes `vexflow.js` grow 100K.